arch/arm/mach-aspeed/ast2700/cpu-info.c

```diff
@@ -83,10 +82,13 @@ void ast2700_print_wdtrst_info(void)
 
 #define SYS_EXTRST		BIT(1)
 #define SYS_SRST		BIT(0)
+/* Sys Scratch reg SCU7FC */
+#define ASPEED_SYS_SCRATCH_7FC 0x12C027FC
 
 void ast2700_print_sysrst_info(void)
 {
 	u32 sys_rst = readl(ASPEED_CPU_RESET_LOG1);
+	writel(sys_rst, ASPEED_SYS_SCRATCH_7FC);
```

Comment:
why are we writing to 0x12C027FC?  You may want to add a sentence about this as a comment or in the commit log.

Reply:
Done